en/ros2/user_guide.md

```diff
@@ -751,6 +751,115 @@ will generate topics under the namespaces:
 
 :::
 
+## PX4 ROS 2 Service Servers
+
+<Badge type="tip" text="PX4 v1.15" />
+
+PX4 uXRCE-DDS middleware supports [ROS 2 services](https://docs.ros.org/en/jazzy/Concepts/Basic/About-Services.html).
+Services are remote procedure calls, from one node to another, that return a result.
+
+A service server is the entity that will accept a remote procedure request, perform some computation on it, and return the result.
+They simplify communication between ROS 2 nodes and PX4 by encapsulating and associating the request and response topics, and ensuring that replies are only returned to the specific requesting user.
+This is much easier that publishing the request, subscribing to the reply, and filtering out any unwanted responses.
+
+The service servers that are built into the PX4 [uxrce_dds_client](../modules/modules_system.md#uxrce-dds-client) module include:
+
+- `/fmu/vehicle_command` (definition: [`px4_msgs::srv::VehicleCommand`](https://github.com/PX4/px4_msgs/blob/main/srv/VehicleCommand.srv).)
+
+  This service can be called by ROS 2 applications to send PX4 [VehicleCommand](../msg_docs/VehicleCommand.md) uORB messages and receive PX4 [VehicleCommandAck](../msg_docs/VehicleCommandAck.md) uORB messages in response.
+
+Details and specific examples are provided in the following sections.
+
+
+### VehicleCommand service
+
+This can be used to send commands to the vehicle, such as "take off", "land", change mode, and "orbit", and receive a response.
+
+The service is available under `/fmu/vehicle_command`, which can be changed through custom namespaces just like the other PX4 topics.
+The service type is defined in [`px4_msgs::srv::VehicleCommand`](https://github.com/PX4/px4_msgs/blob/main/srv/VehicleCommand.srv) as:
+
+```txt
+VehicleCommand request
+---
+VehicleCommandAck reply
+```
+
+Users can make service requests by sending [VehicleCommand](../msg_docs/VehicleCommand.md) messages, and receive a [VehicleCommandAck](../msg_docs/VehicleCommandAck.md) message in response.
+The service ensures that only the `VehicleCommandAck` reply generated for the specific request made by the user is sent back.
+
```

Comment:
Not every command can be sent - in fact most can't. Should/can we list the subset that will work?

Reply:
The ROS 2 node that is making the service request is treated as an internal PX4 module (unless https://github.com/PX4/px4_msgs/blob/912244a6e3a64324f01086569ac9805df854f5d9/msg/VehicleCommand.msg#L195 is set to true). Therefore the user can use any command.

As far as I know, the only commands that has no effect are the ones that deal with PX4 parameters (you cannot set, get, list) parameters.